### PR TITLE
[ANT-302] Replace s10s with s21s

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR
https://jira.unity3d.com/browse/ANT-302

- [x] Compare results with nightly from same base commit + List findings in [this document](https://docs.google.com/document/d/1uyLlcf_0TZ4PEPJUJEaxYwJ9USdH33aDtaRF5UFLOAM/edit)

---

### CODE REVIEW

**Comment by theo-pnv** [comment]
Comparison of failing tests with master: https://docs.google.com/document/d/1uyLlcf_0TZ4PEPJUJEaxYwJ9USdH33aDtaRF5UFLOAM/edit?usp=sharing
All failing jobs are matching master (from november 26th), or are not run on master.

File: .yamato/_abv.yml [hunk 1]
```diff
@@ -7,6 +7,54 @@
 
 ---
 
+TMP_ALL_S21_trunk:
```
Comment (theo-pnv) [inline .yamato/_abv.yml:10]: TODO before merging: Remove TMP job

**Review by tomzig16 — APPROVED** [review]

**Review by sophiaaar — APPROVED** [review]

**Comment by theo-pnv** [comment]
On 10xx and 7xx we are still using hikeys (I think), since we're using `Unity::mobile::android`. Is that intenteded/is it planed to replace these with samsung devices (S21s?) @tomzig16

**Comment by tomzig16** [comment]
> I'm planning the backports for this PR, and realized on 10xx and 7xx we are still using hikeys (I think), since we're using `Unity::mobile::android`. Is that intenteded/is it planed to replace these with samsung devices (S21s?) @tomzig16

I had left hikeys as we used to have small amount of the devices at the time of making the PR. Now if we have enough s21s we could move, but I believe there might be a reference image failure or two. When I was moving from hikeys it was a failure in terrain test on vulkan due to small rasterization difference which was solved by updating reference images so I'd expect something similar on these backports. @theo-pnv

**Pull request merged**